apollo-audit/apollo-audit-impl/src/main/java/com/ctrip/framework/apollo/audit/controller/ApolloAuditController.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2023 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.audit.controller;
+
+import com.ctrip.framework.apollo.audit.api.ApolloAuditLogApi;
+import com.ctrip.framework.apollo.audit.dto.ApolloAuditLogDTO;
+import com.ctrip.framework.apollo.audit.dto.ApolloAuditLogDataInfluenceDTO;
+import com.ctrip.framework.apollo.audit.dto.ApolloAuditLogDetailsDTO;
+import java.util.Date;
+import java.util.List;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/apollo/audit")
+public class ApolloAuditController {
+
+  private final ApolloAuditLogApi api;
+
+  public ApolloAuditController(ApolloAuditLogApi api) {
+    this.api = api;
+  }
+
+  @GetMapping("/logs")
+  public List<ApolloAuditLogDTO> findAllAuditLogs(int page, int size) {
```

Comment:
Audit log is sensitive,

can we add annotation(just annotation) 
```
@PreAuthorize(value = "@permissionValidator.isSuperAdmin()")
```
for the portal?

see com.ctrip.framework.apollo.portal.controller.SystemInfoController#getSystemInfo

Reply:
so i guess we have to add spring-security dependency.
I think we can use properties to make it only affects in Portal

Reply:
```java
@PreAuthorize(value = "@apolloAuditLogQueryApiPreAuthorizer.hasQueryPermission()")
```

add this instead, and create spi ApolloAuditLogQueryApiPreAuthorizer, defalut impl returns true. We can implement this spi for portal uses! such as return  permissionValidator.isSuperAdmin()

Reply:
`public class ApolloAuditLogQueryApiDefaultPreAuthorizer implements
    ApolloAuditLogQueryApiPreAuthorizer {

  @Override
  public boolean hasQueryPermission() {
    return true;
  }
}`

Reply:
```java
@Component
public class ApolloAuditLogQueryApiPortalPreAuthorizer implements
    ApolloAuditLogQueryApiPreAuthorizer {
  
  private final PermissionValidator permissionValidator;

  public ApolloAuditLogQueryApiPortalPreAuthorizer(PermissionValidator permissionValidator) {
    this.permissionValidator = permissionValidator;
  }

  @Override
  public boolean hasQueryPermission() {
    return permissionValidator.isSuperAdmin();
  }
}
```

and this at portal!